pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -2594,6 +2594,7 @@ package-maintainers:
   cdepillabout:
     - pretty-simple
     - purescript
+    - spago
```

Comment:
@peti `spago` isn't actually on Hackage (instead I pull it from github), so is it okay to add spago to the list of packages I maintain here?  Will anything bad happen if I add myself here?

Is this list strictly only for packages that are on Hackage?

Reply:
> Is this list strictly only for packages that are on Hackage?

The only effect this change will have is that `hackage2nix` will refuse configuration files that mark this package as broken (since it has a maintainer that is going to fix the build, eventually).